backend/s3/s3.go

```diff
@@ -1684,11 +1684,8 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	if err != nil {
 		return nil, errors.Wrap(err, "s3: upload cutoff")
 	}
-	if opt.ACL == "" {
-		opt.ACL = "private"
-	}
 	if opt.BucketACL == "" {
-		opt.BucketACL = opt.ACL
+		opt.BucketACL = "private"
```

Comment:
This isn't quite the same logic as before...

Before if ACL == "something" and "BucketACL" == "" then BucketACL becomes "something"

After if ACL == "something" and "BucketACL" == "" then BucketACL becomes "private"

Now whether that is the correct logic or not I don't know!

Reply:
I'm not sure what would be a good default ACL to apply to a new bucket if a bucket ACL isn't supplied, however I guess this has the same issue, i.e. you can't create an ACL on a bucket in a system that doesn't support ACLs or has ACLs disabled, so we should allow not setting an ACL.